src/utils/operations/formatAmount.ts

```diff
@@ -0,0 +1,143 @@
+import { TokenTransfer } from '@multiversx/sdk-core';
+import BigNumber from 'bignumber.js';
+import { DECIMALS, DIGITS, stringIsInteger, ZERO } from 'lib/sdkDappUtils';
+import { pipe } from './pipe';
+
+BigNumber.config({ ROUNDING_MODE: BigNumber.ROUND_FLOOR });
```

Comment:
should be done inside function as a local bigNumber see stringIsInteger or stringIsFloat